src/Compilers/VisualBasic/Portable/Binding/Binder_Lookup.vb

```diff
@@ -1239,13 +1273,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 If symbol Is Nothing Then
                     ' Match the native compiler which reports ERR_XmlFeaturesNotAvailable in this case.
                     Dim useSiteError = ErrorFactory.ErrorInfo(ERRID.ERR_XmlFeaturesNotAvailable)
-                    singleResult = New SingleLookupResult(LookupResultKind.NotReferencable, binder.GetErrorSymbol(name, useSiteError), useSiteError)
+                    singleResult = New SingleLookupResult(LookupResultKind.NotReferencable, Binder.GetErrorSymbol(name, useSiteError), useSiteError)
                 Else
                     Dim reduced = New ReducedExtensionPropertySymbol(DirectCast(symbol, PropertySymbol))
                     singleResult = binder.CheckViability(reduced, arity, options, reduced.ContainingType, useSiteDiagnostics)
                 End If
 
-                lookupResult.MergePrioritized(singleResult)
+                If lookupResult Is Nothing And singleResult.Kind <> LookupResultKind.Empty Then
```

Comment:
>And [](start = 43, length = 3)

AndAlso?

Reply:
💭 The branch didn't seem to offer anything here